ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aro-4.13-nightly-x86.yaml

```diff
@@ -53,7 +53,7 @@ tests:
     - chain: openshift-qe-node-density-heavy
     workflow: openshift-qe-installer-azure-aro
 - as: perfscale-aro-control-plane-6nodes-periodic
-  cron: 0 12 * * 4
+  cron: 0 8 * * 4
```

Comment:
should we lower the cadence of this since its 4.13?

Reply:
Roshni said to run once a week.